src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -50,5 +63,9 @@ public CacheOptions(bool useSharedCache)
         /// </remarks>
         public bool UseSharedCache { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public IEnumerable<string> ExtraTokenParamsToCache { get; set; }
```

Comment:
I recommend using `IReadonlyList` instead of `IEnumerable`. It's readonly and operations such as "Count" are guarateed to be fast, whereas IEnumerable.Count() is more complicated.